js/app.js

```diff
@@ -1,2 +1,179 @@
 'use strict';
+//Define how many products to show at once.
+const NUM_OF_PRODUCTS = 3;
 
+// Define how many rounds there are and the current round to increment for every product click
+const TOTAL_ROUNDS = 25;
+let round = 0;
+
+// Constructor function to create a const  = new product
+function Product(name, src) {
+  this.name = name;
+  this.src = src;
+  this.clicks = 0;
+  this.views = 0;
+}
+
+// Create Products for all included images
+const bag = new Product('bag', '/img/bag.jpg');
+const banana = new Product('banana', '/img/banana.jpg');
+const bathroom = new Product('bathroom', '/img/bathroom.jpg');
+const boots = new Product('boots', '/img/boots.jpg');
+const breakfast = new Product('breakfast', '/img/breakfast.jpg');
+const bubblegum = new Product('bubblegum', '/img/bubblegum.jpg');
+const chair = new Product('chair', '/img/chair.jpg');
+const cthulhu = new Product('cthulhu', '/img/cthulhu.jpg');
+const dogDuck = new Product('dog-duck', '/img/dog-duck.jpg');
+const dragon = new Product('dragon', '/img/dragon.jpg');
+const pen = new Product('pen', '/img/pen.jpg');
+const petSweep = new Product('pet-sweep', '/img/pet-sweep.jpg');
+const scissors = new Product('scissors', '/img/scissors.jpg');
+const shark = new Product('shark', '/img/shark.jpg');
+const sweep = new Product('sweep', '/img/sweep.png');
+const tauntaun = new Product('tauntaun', '/img/tauntaun.jpg');
+const unicorn = new Product('unicorn', '/img/unicorn.jpg');
+const waterCan = new Product('water-can', '/img/water-can.jpg');
+const wineGlass = new Product('wine-glass', '/img/wine-glass.jpg');
+
+const productsArray = [
+  bag,
+  banana,
+  bathroom,
+  boots,
+  breakfast,
+  bubblegum,
+  chair,
+  cthulhu,
+  dogDuck,
+  dragon,
+  pen,
+  petSweep,
+  scissors,
+  shark,
+  sweep,
+  tauntaun,
+  unicorn,
+  waterCan,
+  wineGlass
+];
```

Comment:
It seems unlikely you'll need to access these by variable name, and if you do it seems to be the same as the Product.name. If you wanted to save a few lines of code you could instantiate the objects in the literal of the array.